micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/convention/JvmMetersConventions.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2025 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.jvm.convention;
+
+import io.micrometer.core.instrument.Tags;
+
+/**
+ * Defines methods to get the various conventions related to JVM metrics.
+ *
+ * @since 1.16.0
+ * @see #DEFAULT the default implementation
+ */
+public interface JvmMetersConventions {
```

Comment:
I updated all of the MeterBinders so they aren't referencing this `JvmMetersConventions` anymore. I'm doubtful it is worth keeping it and `MicrometerJvmMetersConventions`. I think I'll remove them and we can add them back if feedback or a clear benefit is demonstrated somehow. They are indirectly used by `OtelJvmMetersConventions` and so I'll need to figure out what makes sense to do there as I add some more tests using that.

Reply:
I've removed the mentioned interfaces now and I've promoted the Otel conventions classes to top-level classes, but I moved them to their own package, as well as the Micrometer conventions to their own package.